### DISCUSSION TITLE
Do not immediately refresh feed for new subscription

### DISCUSSION THREAD

**Discussion by cobratbq** [jonjomckay/fritter#286] [Ideas]
I have a habit of occasionally subscribing to a new account while reading through my twitter feed. The current behavior of Fritter is to refresh the feed -- and consequently scrolling back to the top thus losing progress -- upon returning, presumably because I have subscribed to a new account.

I want to propose to have to have the account show up only after a manual refresh (either drag-down/button refresh or because of tab switching), so that one can continue reading the feed without losing position/progress. Presumably, if the user just subscribed to some account, they would have already checked out that account, so it is not critical to have the new account immediately included in the feed.

**Comment by ghost** [comment]
This discussion can be closed as feed support has been removed
